src/styles/images/index.md.njk

```diff
@@ -8,53 +8,144 @@ layout: layout-pane.njk
+
+Some examples of illustrations from services:
+
+## Icons
+
+Avoid using icons in most cases. People can understand icons to mean different things, so they can be confusing.
+
+Icons can be more useful in case working systems, where users are familiar with the interface and return to it frequently. In this context, they can help users to scan pages more quickly. In most cases it’s still helpful to include a text label alongside any icons.
+
+Do not use a single icon to represent more than one thing. For example, the search icon (magnifying glass) should only be used for search functionality, and not also for screen magnification.
+
+## Avoid images that contain text
+
+Use written content instead of an image that contains text, wherever possible.
+
+Any text inside an image will be difficult to read, especially for users that need to read text using:
+- screen readers
+- copy and paste
+- highlight
+- resize
+- recolour
+
+However, text can sometimes be an inseparable part of an image, especially in photography. Examples of this might be a photo with a signpost in it, or a company's name on the side of a building.
+
+If you choose to use an image that contains text, you should also include written content nearby that conveys the same meaning. This means the information is available to everyone
+
+Only use alt text as a last resort, when you need to use an image with text that cannot be removed, replaced or supplemented by written content. Be sure to follow the guidance on this page about writing good alt text.
+
+### Avoid images of text
+
+Images of text are usually unnecessary. Where possible, you should replace them with alternatives.
+
+Examples of unnecessary images could include:
+- a post from social media
+- an excerpt from a document
+- a screenshot from a presentation
+
+If you do use an image of text, remember to have written content nearby that conveys the same meaning.
+
+#### Exceptions
+
+There are specific types of images where the exact appearance of the text in the image is essential to its purpose. For example, a logo or a handwritten memo.
+
+These types of images must include alt text, but equivalent written content nearby is not required.
+
+However, you should still try to replace it when possible.
+
+### Possible alternatives
+
+If it's not practical to avoid using an image that contains text (and replace it with written content), there’s a few other ways to replace it.
+
+In any case, make sure that the contrast ratio of text colour and all portions of the image that overlap the text [meets level AA of the Web Content Accessibility Guidelines (WCAG 2.1)](https://www.w3.org/TR/WCAG21/#contrast-minimum).
+
+#### Show HTML text over an image
+
+You could place HTML text so that it shows over the image.
+
+You’ll still need to make sure there is enough contrast, between the text colour and all portions of the image that overlap the text, on all screen sizes.
```

Comment:
Suggestion to add...

> on all screen sizes

and zoom levels.

Reply:
```suggestion
You’ll still need to make sure there is enough contrast, between the text colour and all portions of the image that overlap the text, on all screen sizes and all zoom levels.
```